.github/workflows/publish-data.sh

```diff
@@ -0,0 +1,59 @@
+# See publish-data.yml
+
+TMP=pub/tmp
+mkdir $TMP
+
+COPY_YEAR=$(date -u +%Y)
+PUB_DATE=$(date -u +%Y-%m-%d)
+
+cat > $TMP/sed-readmes.txt << eof
+s/COPY_YEAR/$COPY_YEAR/
+s/PUB_DATE/$PUB_DATE/
+s/PUB_STATUS/draft/
+s/UNI_VER/$UNI_VER/
+s/EMOJI_VER/$EMOJI_VER/
+s/TR10_REV/$TR10_REV/
+s%PUBLIC_EMOJI%Public/draft/emoji%
+s%PUBLIC_UCD%Public/draft/UCD%
+eof
+
+mkdir $TMP/UCD
+cp -R unicodetools/data/ucd/dev $TMP/UCD/ucd
+mv $TMP/UCD/ucd/version-ReadMe.txt $TMP/UCD/ReadMe.txt
+rm -r $TMP/UCD/ucd/Unihan
+
+if [ "$RELEASE_PHASE" = "Dev" ]; then
```

Comment:
I would normally go with bash conditional expressions (with `[[` `]]`) rather than the `test` command (aka `[`).

@markusicu, I am unsure about our bash style here, which one do we prefer?

Reply:
I wasn’t familiar with the difference between `[` and `[[`, but now after some reading I do prefer `[[` as it offers an experience similar to modern languages. Made the change.